spark/utilities/helpers/vanilla/isElementVisible.js

```diff
@@ -0,0 +1,12 @@
+/* global document */
+
+const isElementVisible = selector => {
```

Comment:
@guern1kn did these two have any tests that need to be brought into NDS?

Reply:
They sure did. Added them.